kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/transaction/TransactionStateManager.java

```diff
@@ -584,7 +584,7 @@ private Either<Errors, Optional<CoordinatorEpochAndTxnMetadata>> getAndMaybeAddT
             if (loadingPartitions.contains(partitionId)) {
                 log.info("TX Coordinator {} partition {} for transactionalId {} is loading",
                         transactionConfig.getTransactionMetadataTopicName(), partitionId, transactionalId);
-                return Either.left(Errors.COORDINATOR_LOAD_IN_PROGRESS);
+                return Either.left(Errors.COORDINATOR_NOT_AVAILABLE);
```

Comment:
Should we add a comment to let others know why we change to `COORDINATOR_NOT_AVAILABLE`?